service/src/main/java/de/starwit/service/jobs/GeometryConverter.java

```diff
@@ -0,0 +1,54 @@
+package de.starwit.service.jobs;
+
+import java.awt.geom.Area;
+import java.awt.geom.Line2D;
+import java.awt.geom.Path2D;
+import java.awt.geom.Point2D;
+
+import de.starwit.persistence.databackend.entity.AnalyticsJobEntity;
+import de.starwit.persistence.databackend.entity.PointEntity;
+import de.starwit.persistence.sae.entity.SaeDetectionEntity;
+
+public class GeometryConverter {
```

Comment:
some method descriptions would be nice

Reply:
What would you like to know about the methods? I created some comments, but I'm unsure if they really bring any value instead of just adding more text.